doc/content.tex

```diff
@@ -1,1349 +0,0 @@
-            \label{tab:models-stat}
-        \end{center}
-    \end{table}
-
-    Statistics produced by each model are listed in table \ref{tab:models-stat}. Models with the lowest one
-    step ahead prediction error are seasonal ARIMA and triple exponential smoothing which is correct for this highly
-    seasonal time series.
-
-    As it was discussed in section \ref{sec:model-quality} AIC criterion should be used for models comparison. From the
-    table \ref{tab:models-stat} it can be seen that AIC and MSE are related, however MSE of double exponential smoothing
-    is $8.3$ times higher than MSE of triple exponential smoothing, whereas AIC of double exponential smoothing is
-    only $1.3$ times higher. From this example it is obvious how AIC discriminates models with more parameters.
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Existing Solutions}
-Before going directly to the design and implementation let's briefly look at existing monitoring solutions and libraries
-for time series modelling.
-
-In the next section not only direct competitors of Hawkular are described but also
-generally available software of this kind. Because Hawkular is still only an upstream project and not deliverable
-product for the customers, thereby Jboss Operations Network (JON) is considered as its alternative.
-
-However, this work does not focus on precise comparison of monitoring applications. It is only mentioned for wider
-overview of this domain.
-
-    %%%%%%%%
-    \section{Monitoring and Management Applications}
-    Out there are many monitoring solutions available. Some of them are offered as service (SaaS) and other need to be
-    deployed in customer's environment.
-
-    For each application are highlighted the most important features and how it differs from its rivals. It is also
-    mentioned if the application offers predictive engine for metric data.
-
-        %%%%%%%%
-        \subsection{New Relic}
-        New Relic was the first monitoring solution offered as SaaS \cite{new-relic}. It focuses on application
-        performance rather than infrastructure view.
-
-        Monitoring of Java applications is done by an agents. Because agent manipulates with bytecode it is able to
-        identify business transactions and monitor for instance utilization of REST endpoints.
-
-        Agents also automatically discover databases connected to the application being monitored. From alert
-        prediction perspective New Relic does not offer any alert prediction or predictive charts.
-
-        %%%%%%%%
-        \subsection{Dynatrace Ruxit}
-        Ruxit is another popular SaaS solution for monitoring middleware applications. Monitoring is also done by agents
-        which instruments bytecode so the application is also able to monitor business transactions. Ruxit differs from
-        others that it offers root cause analysis rather than firing multiple alerts \cite{ruxit}. It directly shows
```

Comment:
that it offers - in offering
cela veta: Ruxit differs from others in offering root cause analysis rather than firing multiple alerts \cite{ruxit}.